### PR TITLE
Fix pool test

### PR DESCRIPTION
**Pull request by xpivarc**
**What this PR does / why we need it**:
NewRandomVMWithDataVolume is requesting 1Gi
of memory. In the case of pool tests this multiplies
by 2 leading to not enough memory in case of
parallel tests.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/libvmi/vm.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ package libvmi
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
```
Comment (dankenigsberg) [inline tests/libvmi/vm.go:25]: At first view, I was not very happy about this additional dependency, but this dependency is real in the production code and in VM cr, so I don't see a way around having it.

**Review by dankenigsberg — APPROVED** [review]

File: tests/libvmi/vm.go (hunk 1 above)
Comment (xpivarc) [reply tests/libvmi/vm.go:25]: It is part of the API, no way around it.

Comment (dankenigsberg) [reply tests/libvmi/vm.go:25]: hence my lgtm ;-)

File: tests/pool_test.go [hunk 2]
```diff
@@ -116,11 +117,24 @@ var _ = Describe("[sig-compute]VirtualMachinePool", decorators.SigCompute, func(
...
 
+		dataVolume := libdv.NewDataVolume(
+			libdv.WithRegistryURLSource(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskCirros)),
+			libdv.WithPVC(libdv.PVCWithStorageClass(sc)),
+		)
+
+		vm := libvmi.NewVirtualMachine(
+			libvmi.New(
+				libvmi.WithDataVolume("disk0", dataVolume.Name),
+				libvmi.WithResourceMemory("100M"),
```
Comment (dankenigsberg) [inline tests/pool_test.go:133]: Do we test anything about the guest? Does it even boot? You may waste even less memory https://github.com/kubevirt/kubevirt/pull/11033/files#diff-005747350099de38fcd22401693fe634b2c4607b4deec581b298efe440b93d2fL183

**Review by dankenigsberg — APPROVED** [review]

**Pull request merged**